new-website/utils/tutorials/contents.md

```diff
@@ -1,71 +0,0 @@
-# Contents
```

Comment:
The markdown should be source of truth. This needs to be editable by non-programmers